src/entity/Author.entity.ts

```diff
@@ -0,0 +1,40 @@
+import { Entity, PrimaryGeneratedColumn, Column, BaseEntity, OneToMany } from "typeorm";
+import { Book } from "./Book.entity";
+import { COLUMN_LENGTH } from '../../constants';
+
+@Entity()
+export class Author extends BaseEntity {
+    @PrimaryGeneratedColumn()
+    id!: number;
+
+    @Column({ type: 'varchar', length: COLUMN_LENGTH.MEDIUM })
```

Comment:
```suggestion
    @Column({ type: 'varchar', length: COLUMN_LENGTH.NUM_255 })
```
Đặt NUM_ Số cụ thể để dễ phân biệt hơn là từ chung từ medium